@types/jsb.d.ts

```diff
@@ -1415,6 +1415,88 @@ declare namespace jsb {
         showExternalOfferInformationDialog: (listener: ExternalOfferInformationDialogListener) => void;
         showInAppMessages: (params: InAppMessageParams, listener: InAppMessageResponseListener) => void;
     }
+
+    export class PlayException {
```

Comment:
Developers might misinterpret PlayException as an exception related to gameplay rather than something specific to Google Play. Should Google-specific content be encapsulated within a googleplay namespace?

Reply:
It is indeed under the google play namespace now. jsb's interface is not open to the public. Developers use google.play.PlayException.
![image](https://github.com/user-attachments/assets/fd962d64-d5c0-4184-a929-0dcd2e4e905e)